From: Chad Monroe <chad@monroe.io>
Date: Fri, 18 Apr 2025 09:44:29 -0700
Subject: [PATCH] mt76: mt7996: increase txq memory limit to 32 MiB 

Prior to this change, both 2G and 6G radios would fall back to the
mac80211 default of 4MB which is not enough for high data rates.

Signed-off-by: Chad Monroe <chad@monroe.io>
---
 mt7996/init.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mt7996/init.c
+++ b/mt7996/init.c
@@ -530,6 +530,7 @@ mt7996_init_wiphy(struct ieee80211_hw *h
 	ieee80211_hw_set(hw, SUPPORTS_MULTI_BSSID);
 
 	hw->max_tx_fragments = 4;
+	wiphy->txq_memory_limit = 32 << 20; /* 32 MiB */
 
 	/* init led callbacks */
 	if (IS_ENABLED(CONFIG_MT76_LEDS)) {
